### DISCUSSION TITLE
Opening/Closing toggleterm

### DISCUSSION THREAD

**Discussion by RizaHKhan** [LunarVim/LunarVim#3284] [General]
Recent release has removed some keybinds. One being opening and closing Toggleterm:

I tried to add:

```
lvim.keys.normal_mode["<C-t>"] = ":ToggleTerm<CR>"

```

But that only opens it but doesn't minimize it. Can someone provide the appropriate configuration, please?

**Comment by RizaHKhan** [comment]
This has apparently been remapped to `<C-\>`.

**Comment by kobzar** [comment]
In stable bruch by default the key was remapped to
Alt + 1  Alt + 2  Alt + 3
Its really more usefult in work.

**Comment by LostInTheLogs** [comment]
`lvim.builtin.terminal.open_mapping = [[<c-t>]]`
